lib/ansible/modules/clustering/kubevirt/kubevirt_raw.py

```diff
@@ -162,7 +162,8 @@ class KubernetesException(Exception):
 
 RAW_ARG_SPEC = {
     'wait': {'type': 'bool', 'default': True},
-    'wait_timeout': {'type': 'int', 'default': 20}
+    'wait_timeout': {'type': 'int', 'default': 20},
+    'merge_type': {'type': 'list', 'choices': ['json', 'merge', 'strategic-merge']}
```

Comment:
Not sure what python convention we should use, but I like to have ',' at the end, so the path is more easy to read next time we add new parameter.

Reply:
And also, what about adding a default?